src/app/components/elements/list-groups/ListGroupFooterBottom.tsx

```diff
@@ -1,16 +1,53 @@
 import React from "react";
 import { ListGroup, ListGroupItem } from "reactstrap";
 import { ExternalLink } from "../ExternalLink";
+import { Link } from "react-router-dom";
+interface FooterLinkProps {
+  linkTo: string;
+  children?: React.ReactNode | string;
+}
+
+const FooterLink = ({ linkTo, children }: FooterLinkProps) => {
+  return (
+    <ListGroupItem className="border-0 px-0 py-0 bg-transparent align-items-stretch">
+      <Link className="footerLink" to={linkTo}>
+        {children}
+      </Link>
+    </ListGroupItem>
+  );
+};
+
+let key = 0;
```

Comment:
this isn't typically best practice, I would set the key to be more sepcific to the link, like `key={accessibility-footer-link"}`

They should be unique, and this key could be duplicated with the second place this is used in an above file.